homeassistant/components/tplink/sensor.py

```diff
@@ -111,11 +111,11 @@ def async_emeter_from_device(
 def _async_sensors_for_device(
```

Comment:
Changes into this platform were necessary in this PR to avoid setting up a connection on child device sensors, as [we check for `parent` to decide if to use a connection or an identifier for the entity](https://github.com/home-assistant/core/pull/117758/files#diff-3039452aefa70a1099b307cc1eea25b60996f064fa974f679a56c732e09919a8R123).